4/main.js

```diff
@@ -0,0 +1,68 @@
+const sum = 0.1 + 0.3;
+console.log(sum.toFixed(1));
```

Comment:
`.toFixed()` is going to return result in a `String`
Need to convert back to number `+sum.toFixed(1)`